.github/workflows/cypress-tests-auth.yml

```diff
@@ -39,7 +39,9 @@ jobs:
         with:
           build: make e2e-build
           wait-on: http://localhost:4040
-          start: make server -- --auth.internal.enabled --auth.internal.signup-enabled
+          start: |
```

Comment:
didn't know you can do multiple lines like this, that's cool https://github.com/cypress-io/github-action#start-multiple-servers

Reply:
Neither did I, found that in that exact marvelous documentation